ztp/resource-generator/entrypoints/generator

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env bash
```

Comment:
This is a big enough script that we should consider some kind of automated test for it.  I don't know if 'unit test' is quite the right thing, but maybe some kind of integration test that runs as part of the CR (and can be run locally) to check the generator functionality?